cfgov/jinja2/v1/_includes/molecules/related-content.html

```diff
@@ -36,7 +36,9 @@ <h2 class="a-heading">
     {{ parse_links(value.paragraph) | safe }}
 
     {% if value.links %}
-        <ul class="m-list m-list__unstyled m-list__links">
+        <ul class="m-list
+                   m-list__unstyled
+                   m-list__links">
```

Comment:
`m-list__unstyled` and `m-list__links` are redundant. See: https://github.com/cfpb/capital-framework/blob/canary/src/cf-typography/src/molecules/list.less#L6-L8

(Also occurs three times in `related-links.html`, once in `related-metadata.less`.)